### PR TITLE
Expose additional values for prometheus configmap-reload in values.yaml

### PR DESCRIPTION
**Pull request by antonosmond**
@mgoodness 

**What this PR does / why we need it**:
The config-reload image used in the prometheus chart allows multiple directories to be monitored to trigger config-reloads so expose `extraArgs` and `extraConfigMapMounts` in values.yaml so this functionality can be used.

This allows us to do things like separating the alert configs from the main prometheus config making them easier to manage and update.

### CODE REVIEW

File: stable/prometheus/values.yaml [hunk 1]
```diff
@@ -184,6 +184,19 @@ configmapReload:
...
+  ## Additional configmap-reload container arguments
+  ##
+  extraArgs: {}
+
+  ## Additional configmap-reload mounts
+  ##
+  extraConfigmapMounts: []
+    # - name: prometheus-alerts
+    #   mountPath: /etc/alerts.d
+    #   configMap: prometheus-alerts
```
Comment (timstoop) [inline stable/prometheus/values.yaml:196]: I may not fully understand how the code works, but it looks like this line from the example is not being parsed by the actual server-deployment.yaml, correct?

**Review by timstoop — CHANGES_REQUESTED** [review]
As you're adding new features, please bump the minor version of Chart.

File: stable/prometheus/values.yaml (hunk 1 above)
Comment (antonosmond) [reply stable/prometheus/values.yaml:196]: Good spot you're absolutely correct! I've missed the bit under volumes e.g. [here](https://github.com/echo-health/charts/blob/master/stable/prometheus/templates/server-deployment.yaml#L148-L152). I'll add that in.

**Comment by antonosmond** [comment]
@timstoop - made the suggested changes. Thanks!

**Comment by antonosmond** [comment]
@timstoop @mgoodness Hold off doing anything with this. I need to split the volumes from the mounts as currently, if for example prometheus-server AND configmap-reload both want to mount the same extraConfigmapMounts, it'll try to create the volume twice and fail. This is because it's creating the volumes based on the mounts [here](https://github.com/echo-health/charts/blob/e15cf015e88c68b2b5bf1e33b1dabd8043e399b5/stable/prometheus/templates/server-deployment.yaml#L148-L152)

**Comment by antonosmond** [comment]
@mgoodness @timstoop closing in favour of https://github.com/kubernetes/charts/pull/4511

**Pull request closed without merging**